cognee-mcp/apps-src/src/design.css

```
@@ -0,0 +1,566 @@
+
+/* ── Result panel ─────────────────────────────────────────────── */
+.result {
+  border-bottom: 1px solid var(--cg-border);
+  background: #fff;
+  padding: 12px 16px 14px;
+  display: flex; gap: 12px;
+  position: relative;
+  animation: fadeUp 250ms var(--cg-ease) both;
+  max-height: 240px; overflow-y: auto;
+}
+@keyframes fadeUp {
```

Stylelint (17.9.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)